src/components/ItemMenu/ItemMenu.js

```diff
@@ -1,43 +1,12 @@
-import React from 'react';
+import React from 'react'
 
-import './ItemMenu.css'
-
-
-class ItemMenu extends React.Component {
-
-  constructor(props){
-    super(props)
-
-    this.state = {
-      isActive: false
-    }
-
-  }
-
-  handleClick = () => {
-    this.setState(state=> ({
-      isActive: !state.isActive
-    }));
-  }
-
-  render() {
-    let key = 0;
-
-    let className = this.props.className;
-
-    return(
-
-      this.props.itemsArr.map(item => {
-
-        (!this.state.isActive) ? className = this.props.className : className = "active-item";
-
-          return(
-          <span className={className} key={key++} onClick={this.handleClick}>{item.name}</span>
+class itemMenu extends React.Component {
```

Comment:
[Use PascalCase for filenames. E.g., ReservationCard.jsx](https://airbnb.io/javascript/react/#naming)